frontend/src/views/SettingsView.vue

```diff
@@ -94,6 +94,7 @@ onMounted(() => {
         "
         :class="{ '!bg-teal-500 !text-white': view === activeView }"
         @click="show(key)"
+        :data-testid=key
```

Comment:
Is there a reason to not using quotes here? I think we always use them even on single word attributes.

```suggestion
        :data-testid="key"
```

Reply:
Oh and the term `key` might be a bit too general as an identifier? But I'm really no expert here, if that is sufficient that's totally fine.